packages/forms/src/styled/text/StyledTextInput.ts

```diff
@@ -131,8 +129,8 @@ const sizeStyles = (props: IStyledTextInputProps & ThemeProps<DefaultTheme>) =>
   const swatchMarginHorizontal = math(
     `${paddingVertical} + ${swatchMarginVertical} - ${paddingHorizontal}`
   );
-  const figureMarginFirst = `auto ${math(`${paddingHorizontal} * 3 / 4`)} auto 0`;
-  const figureMarginLast = `auto 0 auto ${math(`${paddingHorizontal} * 3 / 4`)}`;
+  const figureMarginFirst = `auto ${math(`${paddingHorizontal} * 1 / 2`)} auto 0`;
```

Comment:
Is the math logic necessary here? I think the margins for these in `default` and `isCompact` sizes can be 8px.

Reply:
I believe @jzempel handled this logic originally. Is it ok to still perform this `math()` operation?

Reply:
It's probably worth retaining the calculated relationship between these spacing properties. But it looks like most instances of `math` could be replaced with interpolated calculations with `px` appended.

Reply:
Although, I think @rossmoody is right that it is no longer worth feeding an unchanging value into this particular calc. It's always meant to be `${props.theme.space.base * 2}`.

Reply:
I've updated some of the `math()` values to use interpolation where possible.

Reply:
With the updated spacing it came out to `${props.theme.space.base * 1.5}px` for me. @rossmoody could you double-check the visuals on this once it's deployed?

Reply:
It looks like the first round of approved logic was slightly off. Moving to 8px.